modules/cli/src/main/scala/scala/cli/commands/HelpOptions.scala

```diff
@@ -0,0 +1,10 @@
+package scala.cli.commands
+
+import caseapp._
+
+case class HelpOptions()
```

Comment:
This should make it 
```suggestion
@HelpMessage("Print help message")
case class HelpOptions()
```

This should give a description to the `help` command in the help message, like
```text
Other commands:
  bsp                                       Start BSP server
  export                                    Export current project to sbt or Mill
  help                                      Print help message
  install completions, install-completions  Installs completions into your shell
  setup-ide                                 Generate a BSP file that you can import into your IDE
  shebang                                   This command is an equivalent of `run`, but it changes the way how
  update                                    Update scala-cli - it works only for installation script
```